src/features/hotspots/settings/discovery/DiscoveryMap.tsx

```diff
@@ -0,0 +1,326 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable react/jsx-props-no-spreading */
+import React, {
+  memo,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react'
+import MapboxGL, {
+  CircleLayerStyle,
+  LineLayerStyle,
+  OnPressEvent,
+  SymbolLayerStyle,
+  Expression,
+} from '@react-native-mapbox-gl/maps'
+import { BoxProps } from '@shopify/restyle'
+import { Animated, Easing, StyleProp } from 'react-native'
+import { omit } from 'lodash'
+import { Theme } from '../../../../theme/theme'
+import Box from '../../../../components/Box'
+import {
+  DiscoveryRequest,
+  DiscoveryResponse,
+  DISCOVERY_DURATION_MINUTES,
+} from '../../../../store/discovery/discoveryTypes'
+import { hotspotsToFeatures } from '../../../../utils/mapUtils'
+import { useColors } from '../../../../theme/themeHooks'
+import { NetworkHotspot } from '../../../../store/networkHotspots/networkHotspotsSlice'
+import sleep from '../../../../utils/sleep'
+import usePrevious from '../../../../utils/usePrevious'
+
+const styleURL = 'mapbox://styles/petermain/ckjtsfkfj0nay19o3f9jhft6v'
```

Comment:
Should we put this in a helper or const file so we can share it? Or maybe just export from the standard Map?